### ISSUE TITLE
Removal of username, password, trustStorePath etc from ConnectOptions

### ISSUE THREAD

**Issue by bwgjoseph** [bwgjoseph/mongoose-vs-ottoman#108]
Hi,

I notice that the recent change to `ConnectOptions` removed quite a number of properties.

Previously, I connect as such

```js
ottoman.connect({
    connectionString: 'couchbase://localhost',
    bucketName: 'testBucket',
    username: 'user',
    password: 'password',
  });
```

Now, I can only connect with 

```js
ottoman.connect('couchbase://localhost/testBucket@user:password');
```

The docs still showing the username, password way of connecting - https://ottomanjs.com/guides/quick-start.html#connecting-to-a-couchbase-bucket

Changes done in this commit https://github.com/couchbaselabs/node-ottoman/commit/6ab72220b3b3a7f71c0b3f8d3321aeb65999c47f#diff-d37b052f6aba3859ca946e577d821bcd46f834043744f0926a6f334e7db60def

How do I do so in the future?

Thanks

**Comment by AV25242** [comment]
Nothing has changed @bwgjoseph  
The code has been refactored to extend  `CouchbaseConnectOptions`

`export interface ConnectOptions extends CouchbaseConnectOptions {`

![image](https://user-images.githubusercontent.com/5351738/132392918-864382f4-6a20-48a2-98a9-f5be929b8185.png)

Things should work as before, please let us know if thats not the case

**Comment by bwgjoseph** [comment]
Yup, I can't.

![image](https://user-images.githubusercontent.com/37484538/132437308-550c61df-bf10-4795-b792-49685dc0a1bb.png)

If you have been using the same `node_modules` in your machine, it will work. Try removing the entire `node_modules` and `package-lock.json` and run `npm i` again.

Not sure if it has something to do with #104.

![image](https://user-images.githubusercontent.com/37484538/132437513-379bb85a-06b0-480f-a2dd-ea74b8138876.png)

**Comment by AV25242** [comment]
Hi @bwgjoseph  we will try fresh on our machines and let you know thank you !

**Comment by AV25242** [comment]
Hello @bwgjoseph  both Eric and I tried but we were successful. 

![Screen Shot 2021-09-10 at 3 51 26 PM](https://user-images.githubusercontent.com/5351738/132917149-5c014fa6-3ea2-46df-939d-cd85c402f8c5.jpg)

At this point we aren't sure what is wrong but we strongly feel its something to do with your configuration.
Can you please confirm which version of Ottoman are you using. We are on `2.0.0-beta.8` 
Can you also confirm the version from the package.json inside node_modules\Ottoman

**Comment by bwgjoseph** [comment]
Thank you Arun and Eric! 

Based on what you mentioned, I went to check, and verify the ottoman version, and it is correct. But I figured out the problem was because of mixed ottoman package installed.

I installed `ottoman-2.0.0-beta.8` on my project, but also had `feathers-ottoman-0.3.1` installed on my project. Thing is that `feathers-ottoman` still hasn't upgrade to `beta.8` (currently on `beta.3` and I think that's the problem where there is `beta.3 and .8` in my project which cause the problem.

Apologies for the confusion. I tested on my brand new project, and only installed `beta.8`, and it does work.
